ambry-account/src/main/java/com/github/ambry/account/HelixAccountService.java

```diff
@@ -144,6 +144,16 @@ public Account getAccountById(short id) {
     return accountInfoMapRef.get().getAccounts();
   }
 
+  @Override
+  public short getContainerIdForLegacyPutPublicBlob() {
```

Comment:
why are these functions and not just static variables in `Account`? Another way of phrasing the question - how does this change across implementations?